ruby26/plan.sh

```diff
@@ -0,0 +1,44 @@
+pkg_name=ruby26
+pkg_origin=core
+pkg_version=2.6.1
+pkg_description="A dynamic, open source programming language with a focus on \
+  simplicity and productivity. It has an elegant syntax that is natural to \
+  read and easy to write."
+pkg_license=("Ruby")
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_source=https://cache.ruby-lang.org/pub/ruby/2.6/ruby-${pkg_version}.tar.gz
+pkg_upstream_url=https://www.ruby-lang.org/en/
+pkg_shasum=17024fb7bb203d9cf7a5a42c78ff6ce77140f9d083676044a7db67f1e5191cb8
+pkg_deps=(core/glibc core/ncurses core/zlib core/openssl core/libyaml core/libffi core/readline)
+pkg_build_deps=(core/coreutils core/diffutils core/patch core/make core/gcc core/sed)
+pkg_lib_dirs=(lib)
+pkg_include_dirs=(include)
+pkg_bin_dirs=(bin)
+pkg_interpreters=(bin/ruby)
+pkg_dirname="ruby-$pkg_version"
+
+do_prepare() {
+  export CFLAGS="${CFLAGS} -O3 -g -pipe"
+  build_line "Setting CFLAGS='$CFLAGS'"
+}
+
+do_build() {
+  ./configure \
+    --prefix="$pkg_prefix" \
+    --enable-shared \
+    --disable-install-doc \
+    --with-openssl-dir="$(pkg_path_for core/openssl)" \
+    --with-libyaml-dir="$(pkg_path_for core/libyaml)"
+
+  make
+}
+
+do_check() {
+  make test
+}
+
+do_install() {
+  do_default_install
+  gem update --system --no-document
```

Comment:
Do we need to lock the gem version to 2.7.8 in Ruby2.6 like we did with other Rubies?  #2235

Reply:
Nope we definitely want to be at 3.0.2 or later now